src/main/java/com/ays/admin_user/service/impl/AdminUserRegisterServiceImpl.java

```diff
@@ -78,7 +78,7 @@ public void register(final AdminUserRegisterRequest registerRequest) {
         }
 
         final AysPhoneNumber phoneNumber = registerRequest.getPhoneNumber();
-        if (adminUserRepository.existsByCountryCodeAndLineNumber(phoneNumber.getCountryCode(), phoneNumber.getLineNumber())) {
+        if (adminUserRepository.existsByCountryCodeAndLineNumber(phoneNumber.getCountryCode().toString(), phoneNumber.getLineNumber().toString())) {
             throw new AysAdminUserAlreadyExistsByPhoneNumberException(phoneNumber);
         }
         log.trace("Admin User Register Request checked successfully!");
```

Comment:
Bu sınıf içerisindeki geliştirmelere ihtiyaç kalmadığı için geri alabilir miyiz?